tests/CMakeLists.txt

```diff
@@ -97,7 +97,7 @@ set(LVGL_TEST_OPTIONS_16BIT
 
 set(LVGL_TEST_OPTIONS_16BIT_SWAP
     -DLV_COLOR_DEPTH=16
-    -DLV_COLOR_16_SWAP=1  

```

Comment:
I think the easiest is for you to make a second commit with the whitespace changes reverted. The reviewer, when approving your PR, can use [squash and merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-pull-request-commits). Alternatively you could rebase/squash and then do a force push to your branch.

Reply:
Sorry, I thought I was sending the changes from #3133 into this commit.

"Problem" is I'm using a git hook that look for trailing whitespaces and fixes them in the staged files just before the commit, so if we want to prevent that I would need to skip that git hook, do the commit and then force the git hook to fix the trailing whitespaces. Or before any change force the git hook on the file I need to work on.

Reply:
Maybe not worth it then. You could do the whitespace fixup in a separate commit and then rebase your branch on master once it lands, but that's up to you. I'm not a project collaborator with PR approve privileges, I'm just a contributor.  Maybe keep the whitespace in this PR and do that only if requested by the collaborator.